### ISSUE TITLE
Visited link does not have enough color contrast on backgrounds that are darker than white

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#62]
regular color #006365
hover color #004D4F

**Comment by Tajana-Somogyi** [comment]
We had to put the same color for regular and hover (darker one) because browsers (for security reasons) do not allow full control for styling visited links and automatically add opacity to the set color. See issue #120
